compose.yaml

```diff
@@ -0,0 +1,35 @@
+services:
+  redis:
+    image: redis:alpine
+    ports:
+      - "6379:6379"
+  api:
+    build:
+      context: .
+      dockerfile: dockerfiles/Dockerfile.api
+    depends_on:
+      - redis
+    ports:
+      - "8000:8000"
+    environment:
+      - BROKER_URL=redis://redis:6379/0
+      - CELERY_RESULT_BACKEND=redis://redis:6379/0
```

Comment:
This needs to be CELERY_RESULT_BACKEND=redis://redis:6379/1, so that we use database 0 for the broker and database 1 for the results backend.